robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -241,4 +241,26 @@ public void run() {
         .resume()
         .visible();
   }
+  
+  public ActivityController<T> configurationChange(final Application application, final Configuration newConfiguration) {
+    final T activity = get();
+    final Configuration currentConfig = activity.getResources().getConfiguration();
+    final int changedBits = currentConfig.diff(newConfiguration);
+    currentConfig.setTo(newConfiguration);
+    
+    // Can the activity handle itself ALL configuration changes?
+    if ((getActivityInfo(application).configChanges & changedBits) == changedBits) {
+      shadowMainLooper.runPaused(new Runnable() {
+        @Override
+        public void run() {
+          ReflectionHelpers.callInstanceMethod(Activity.class, component, "onConfigurationChanged",
+            ClassParameter.from(Configuration.class, newConfiguration));
+        }
+      });
+    } else {
+      Shadows.shadowOf(activity).recreateWithConfigChanges(changedBits);
```

Comment:
Not sure how that would work. We need to change the `changingConfiguration` flags so method calls to `isChangingConfigurations` and `getChangingConfiguration` for that specific activity will return the proper values, and making that flag public would be worst in my opinion...

Am I missing something?